spanner/spanner_snippets/snippet_test.go

```diff
@@ -246,4 +289,50 @@ func TestSample(t *testing.T) {
 	assertContains(t, out, "4 Venue 4")
 	assertContains(t, out, "19 Venue 19")
 	assertContains(t, out, "42 Venue 42")
+
+	out = runCommand(t, "createbackup", dbName)
+	assertContains(t, out, "Created backup [")
+	assertContains(t, out, "/backups/my-backup] from database")
+	out = runCommand(t, "cancelbackup", dbName)
+	assertContains(t, out, "Backup cancelled.")
+	out = runCommand(t, "listbackups", dbName)
+	assertContains(t, out, "/backups/my-backup [READY] - ")
+	assertContains(t, out, "Backup count: ")
+	out = runCommand(t, "listbackupsbyname", dbName)
+	assertContains(t, out, "/backups/my-backup [READY] - ")
+	assertContains(t, out, "Backup count: ")
+	out = runCommand(t, "listsmallbackups", dbName)
+	assertContains(t, out, "/backups/my-backup [READY] - ")
+	assertContains(t, out, "Backup count: ")
+	out = runCommand(t, "listnewbackups", dbName)
+	assertContains(t, out, "/backups/my-backup [READY] - ")
+	assertContains(t, out, "Backup count: ")
+	out = runCommand(t, "listinstancebackups", dbName)
+	assertContains(t, out, "/backups/")
+	out = runCommand(t, "listbackupoperations", dbName)
+	assertContains(t, out, "/operations/")
+	assertContains(t, out, "Backup operation count: ")
+	out = runCommand(t, "listdatabaseoperations", dbName)
+	assertContains(t, out, "/operations/")
+	assertContains(t, out, "Database operation count: ")
+	out = runCommand(t, "updatebackup", dbName)
+	assertContains(t, out, "Updated backup [my-backup]")
+	out = runCommand(t, "restorebackup", restoreDbName)
+	assertContains(t, out, "Restored backup [")
+	assertContains(t, out, "/backups/my-backup]")
+
+	// Wait for database to finish optimizing - cannot delete a backup if a database restored from it
```

Comment:
Can we make a function for this?

Reply:
similar to `assertContains := func(t *testing.T, out string, sub string) {....` or one outside the test and pass in the admin client, context etc.?

Reply:
Yes, that's correct. It can be a specific function, something like "WaitForDBReadyOptimizing". I expect here to look more neat for each command test. So it will be easier to read.

Reply:
I've made the function separate in 3e44508